odoo/tools/translate.py

```diff
@@ -1605,6 +1605,11 @@ def _get_translation_upgrade_queries(cr, field):
                       JOIN "{Model._table}" m ON t.res_id = m.id
                       JOIN website w ON m.website_id = w.id
                       JOIN res_lang l ON w.default_lang_id = l.id
+                    UNION
+                    SELECT t.res_id, m."{field.name}", t.value, t.noupdate, 'en_US'
+                      FROM t
+                      JOIN "{Model._table}" m ON t.res_id = m.id
+                     WHERE m.website_id IS NULL
```

Comment:
```suggestion
                      SELECT t.res_id, m."{field.name}", t.value, t.noupdate,  COALESCE(l.code, 'en_US') AS code
                      FROM t
                      JOIN "{Model._table}" m ON t.res_id = m.id
                      LEFT JOIN website w ON m.website_id = w.id
                      LEFT JOIN res_lang l ON w.default_lang_id = l.id
```

could you check if this one has the same result and better performance?

Reply:
Hi, @jip-odoo is it faster?

Reply:
@HydrionBurst  Thank you for reviewing the PR
yes 
```sql_query
UNION
                    SELECT t.res_id, m."{field.name}", t.value, t.noupdate, 'en_US'
                      FROM t
                      JOIN "{Model._table}" m ON t.res_id = m.id
                     WHERE m.website_id IS NULL
```
this is a little bit faster